src/kaleidoscope/driver/BaseLeds.h

```diff
@@ -0,0 +1,51 @@
+/* -*- mode: c++ -*-
+ * BaseLeds -- LED hardware driver base class for Kaleidoscope
```

Comment:
I'm not quite mind-melded enough yet to be suggesting good names, but "BaseKeyscanner" and "BaseLeds" don't have parity. is it a BaseLedDriver"?

Reply:
`BaseLedDriver` sounds good to me. It's certainly better than `BaseLeds`, so I'll go with the new name 'till we find an even better one.